README.md

```diff
@@ -1 +1,263 @@
-# angular-inheriting-or-isolate-scope-readme
\ No newline at end of file
+# Inheriting or isolating scope
+
+## Overview
+
+So far, our directives we've made aren't really that useful. We're not doing anything special - let's change that!
+
+## Objectives
+
+- Describe scope inheritance
+- Describe scope isolate
+- Create an inherited scope
+- Create an isolate scope
+- Bind data to the scope
+- Access inherited and isolate scope properties
+
+## Scope inheritance
+
+Now, one cool things about our directives is that they can display dynamic data. For instance, we might want to have a Twitter card to display a users Twitter handle, along with a link to follow said user.
+
+We can do this exactly like we've done in our views so far - using `{{}}`.
+
+Note: now that we've started using more HTML in our directives, instead of just passing a string, we pass in a multi-line array, joined by an empty string. This means we can have a multi-lined template without having to worry about string concatenation.
```

Comment:
Is this pattern preferred to using a separate template file? Or is it just for this example?